go.mod

```diff
@@ -0,0 +1,27 @@
+module github.com/tektoncd/triggers
+
+go 1.13
+
+require (
+	github.com/golang/protobuf v1.3.4
+	github.com/google/cel-go v0.3.2
+	github.com/google/go-cmp v0.4.0
+	github.com/google/go-github v17.0.0+incompatible
+	github.com/google/go-querystring v1.0.0 // indirect
+	github.com/gorilla/mux v1.7.4
+	github.com/tektoncd/pipeline v0.10.1
+	github.com/tidwall/gjson v1.5.0 // indirect
+	github.com/tidwall/sjson v1.0.4
+	go.uber.org/zap v1.14.0
+	golang.org/x/xerrors v0.0.0-20191204190536-9bdfabe68543
+	gomodules.xyz/jsonpatch/v2 v2.1.0 // indirect
+	google.golang.org/genproto v0.0.0-20200227132054-3f1135a288c9
+	gopkg.in/yaml.v2 v2.2.8
+	k8s.io/api v0.17.3
```

Comment:
Those needs to have a `replace` and target 1.13.8 for current master.

Reply:
```
replace (
	k8s.io/api => k8s.io/api v0.16.5
	k8s.io/apimachinery => k8s.io/apimachinery v0.16.5
	k8s.io/client-go => k8s.io/client-go v0.16.5
)
```
right? copy from https://github.com/tektoncd/pipeline/blob/master/go.mod#L71

Reply:
well this won't work with triggers because triggers still depends on 0.10.x of pipeline, so the version is 0.13.8 I think — but that version might not support go modules as well as 0.16.x does :sweat: worth a try though :wink: